README.md

```diff
@@ -1,47 +1,71 @@
 # wsus_client
 
-#### Table of Contents
+[`auto_update_option`]: #auto_update_option
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
-3. [Setup - The basics of getting started with wsus_client](#setup)
-  * [What wsus affects](#what-wsus_client-affects)
-  * [Beginning with wsus](#beginning-with-wsus_client)
-4. [Usage - Configuration options and additional functionality](#usage)
-  * [Scheduled Auto Update](#scheduled-auto-update)
-  * [Enabling Status Server](#enabling-status-server)
-  * [Enabled parameters](#enabled-parameters)
-5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
-6. [Limitations](#limitations)
+[`detection_frequency_hours`]: #detection_frequency_hours
 
-## Overview
+[`enable_status_server`]: #enable_status_server
+[enabled parameter]: #enabled-parameters
 
-This tool is used to configure WSUS (Windows Server Update Service) settings on client nodes to point to a particular host and settings pertinent to scheduling and installing patches passed down from your WSUS server.
+[`purge_values`]: #purge_values
 
-## Module Description
+[TechNet]: https://technet.microsoft.com/en-us/library/dd939844.aspx
+[WSUS server]: https://technet.microsoft.com/en-us/library/hh852338.aspx
 
-This module can be used to configure agent nodes to point to a WSUS Server for patches instead of Windows Update servers. But also can be used to schedule updates and configure if the user should be able to manage them or even if auto update is enabled.
+[`reboot_relaunch_timeout_minutes`]: #reboot_relaunch_timeout_minutes
+[`reboot_warning_timeout_minutes`]: #reboot_warning_timeout_minutes
 
-## Setup
+[`reschedule_wait_time_minutes`]: #reschedule_wait_time_minutes
 
-Install this module with the following command:
+[`scheduled_install_day`]: #scheduled_install_day
+[`scheduled_install_hour`]: #scheduled_install_hour
+[`server_url`]: #server_url
 
-~~~
-$ puppet module install [--modulepath <path>] puppetlabs/wsus_client
-~~~
+[`target_group`]: #target_group
+
+#### Table of Contents
+
+1. [Module Description - What is the wsus_client module, and what does it do?](#module-description)
+  * [What wsus_client affects](#what-wsus_client-affects)
+2. [Setup - The basics of getting started with wsus_client](#setup)
+  * [Beginning with wsus_client](#beginning-with-wsus_client)
+3. [Usage - Configuration options and additional functionality](#usage)
+  * [Schedule automatic updates](#schedule-automatic-updates)
```

Comment:
I think the two things you are looking for @cyberious are moved to other sections because they were less like usage examples of a thing you can do. So Enabled Parameters got moved down, under Reference, and Enabling Status Server is explained in the relevant parameter. 

Enabled Parameters was moved because it was about "how something works" rather than "how to do something". I think with the Status Server, it wasn't really clear to us why the usage example was needed. The parameter seemed pretty straightforward; is there more to it than that? Like, is there something special the user might be trying to accomplish with this?